src/eddymotion/estimator.py

```diff
@@ -75,23 +76,18 @@ def fit(
             Number of parallel jobs.
         seed : :obj:`int` or :obj:`bool`
             Seed the random number generator (necessary when we want deterministic
-            estimation). If an integer, the value is used to initialize the
-            generator; if ``True``, the arbitrary value of ``20210324`` is used
-            to initialize it.
+            estimation). See :func:`_sort_dwdata_indices`.
         Return
         ------
         affines : :obj:`list` of :obj:`numpy.ndarray`
             A list of :math:`4 \times 4` affine matrices encoding the estimated
             parameters of the deformations caused by head-motion and eddy-currents.
 
         """
-        align_kwargs = align_kwargs or {}
 
-        _seed = None
-        if seed or seed == 0:
-            _seed = 20210324 if seed is True else seed
+        align_kwargs = align_kwargs or {}
 
-        rng = np.random.default_rng(_seed)
+        index_order = _sort_dwdata_indices(seed, len(dwdata))
```

Comment:
This is way better than it was - thanks. For the future, I think we should make this a generator that allows a number of ordering strategies (#111)

Reply:
:+1: IMO the strategies should be put into a `utils.py` module, as they may be of interest outside the estimator. Would also allow the estimator class/module become less complex/more readable.

Reply:
Totally agree - let's follow up on #111. Can you quickly draft your thoughts there?

Reply:
Will do.